agent/tracestate.md

```diff
@@ -0,0 +1,39 @@
+# Trace state
+
+The Hypertrace agents encode vendor specific information in [tracestate header](https://www.w3.org/TR/trace-context/#tracestate-header).
+
+## Format
+
+The tracestate format is `vendorname1=opaqueValue1,vendorname2=opaqueValue2`. The value
+is up to 256 characters long and contains ASCII from `0x20` to `0x7E` excluding `,` and `=`. For more
+detailed information see the [specification](https://www.w3.org/TR/trace-context/#value).
+
+The Hypertrace vendor name is `hypertrace`.
+
+### Value
+
+The Hypertrace tracestate vendor value can encode multiple values. The values are separated by `|`. 
+
+An example:
+* `tracestate: somevendor=opaqueValue1,hypertrace=key1:val1|key2:val2`.
+
+#### Capture
+
+The capture instructs agent what data should be captured. The agent MUST set the capture value
+if the value was not previously present.
+
+Value key: `cap`
+
+Possible values:
+* `0` - capture all request/response headers and payloads
```

Comment:
should we change the values here?

`cap == 1` reads more naturally that data capture has been enabled.

Reply:
+1

Reply:
my original thinking was that the "advanced" mode is the default.